### PR TITLE
Git-hooks blog article

### PR DESCRIPTION
**Pull request by juliahayward**

### CODE REVIEW

File: _posts/2020-05-18-using-git-hooks-with-flyway.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+developers work on the same database migration code at the same time.
+
+## Problem one: Numbering conflicts
+
+Let's say you've been asked to add a new feature to your application, and this needs a new column to be added to the
+database. The database schema is currently at v4; so the natural thing to do is create a script `V5__MyNewColumn.sql`
+and check it into source control. So far, all is good.
+
+Meanwhile, one of your colleagues is also working on a new feature and has added a script `V5__MyNewTable.sql`. As far
+as he's concerned, all appears good too. When the two sets of changes make it into the master branch ready
```
Comment (Lyeeedar) [inline _posts/2020-05-18-using-git-hooks-with-flyway.md]: Should we be using gender neutral pronouns? e.g. As far as they're concerned

File: _posts/2020-05-18-using-git-hooks-with-flyway.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+for testing and deployment, source control will see them as two unrelated files and therefore will not detect any
+problem. However, Flyway will see two files with the same schema version number and will fail validation. 
+
+## Problem two: Ordering conflicts
+
+So you decide to sidestep the above problem by agreeing in advance that you will write script `V5__MyNewColumn.sql` and 
+your colleague
```
Comment (Lyeeedar) [inline _posts/2020-05-18-using-git-hooks-with-flyway.md]: Newline in middle of sentence

File: _posts/2020-05-18-using-git-hooks-with-flyway.md (hunk 1 above)
Comment (juliahayward) [reply _posts/2020-05-18-using-git-hooks-with-flyway.md]: :can-of-worms:

I think whatever we go with it should be consistent throughout; there's quite a few "he" which would also need to change.

Comment (juliahayward) [reply _posts/2020-05-18-using-git-hooks-with-flyway.md]: Actually, just side-stepped it by making colleagues plural (so no pronoun issue).

**Pull request merged**